### ISSUE TITLE
"Edit with NeoVim"

### ISSUE THREAD

**Issue by Leandros** [equalsraf/neovim-qt#372]
Similar to GVim, it would be great if NeoVim-Qt had a shell shortcut to quickly open an instance of neovim, or edit a new file inside an already open instance.

**Comment by equalsraf** [comment]
It should be the same as with regular nvim, for example have you seen https://github.com/mhinz/neovim-remote ?

**Comment by prabirshrestha** [comment]
There is an issue opened for this already in neovim at https://github.com/neovim/neovim/issues/6649

**Comment by Leandros** [comment]
@equalsraf Yep, I've seen neovim-remote. Unfortunately, it seems to be UNIX only, hence I couldn't try it yet.

**Comment by equalsraf** [comment]
@Leandros ah ok. It might need some tweeking but I think it can work - In windows the neovim server is listening on named pipes instead of unix sockets so /tmp/nvimsocket is not valid.

I'm not on windows so I cannot test this right now. The hard part is composing a valid listen address (see :h serverstart()) but `\\.\pipe\nvim-pipe-1234` should be valid i.e.

**Comment by jgehrig** [comment]
It is possible to manually add the context menu entry.

See the comment from Issue #336:
>Here's the guide for vim; you'll want to edit the .reg files to use neovim instead:
http://vim.wikia.com/wiki/Adding_Vim_to_MS-Windows_File_Explorer_Menu

I should add some documentation in the Wiki.

**Comment by younger-1** [comment]
seems solved. see [here](https://github.com/neovim/neovim/pull/17439/)
